### ISSUE TITLE
Avoid compute in datasets

### ISSUE THREAD

**Issue by TomAugspurger** [dask/dask-ml#265]
https://github.com/dask/dask-ml/blob/d5801584d092d8f13f1b38aaf4da5dc3caa6a213/dask_ml/datasets.py#L332 isn't great, especially in settings like Hyperband #221, that are using the distributed scheduler.

We could probably replace

```python
    rng = dask_ml.utils.check_random_state(random_state)

```

with 

```python
    rng = sklearn.utils.check_random_state(random_state)
```

and draw

1. `informative_idx`
2. random data to seed the `dask.array.RandomState` that is eventually used to generate the large random data.

**Comment by dma092** [comment]
Is this still open?

**Comment by TomAugspurger** [comment]
Yes, I think so. There may be a draw_seed in utils may help.

**Comment by dma092** [comment]
Can I work on it?

**Comment by TomAugspurger** [comment]
That would be a great. The docs have contributing guidelines.

**Comment by stsievert** [comment]
> The docs have contributing guidelines.

https://ml.dask.org/contributing.html

**Comment by dma092** [comment]
It seems to me that all the   tests  in test_datsets passes after just commenting 
` informative_idx, beta = dask.compute(informative_idx, beta) ` .
What do you think?

**Comment by TomAugspurger** [comment]
If all the tests pass then that should be fine.

**Comment by dma092** [comment]
> If all the tests pass then that should be fine.

Are you talking about only the tests in test_datasets.py?

**Comment by TomAugspurger** [comment]
I meant the entire test suite, since other tests use it.

**Comment by ayush947** [comment]
I want to contribute in this project
